internal/coreinternal/attraction/attraction.go

```diff
@@ -316,7 +316,11 @@ func (ap *AttrProc) Process(ctx context.Context, logger *zap.Logger, attrs pcomm
 			if !found {
 				continue
 			}
-			attrs.Update(action.Key, av)
+			val, found := attrs.Get(action.Key)
+			if !found {
+				continue
+			}
+			av.CopyTo(val)
```

Comment:
Nit: just for brevity?
```suggestion
			if found {
				av.CopyTo(val)
			}
```

Reply:
I don't like this. The code should be written with the "check conditions" first, instead of "if cond do foo"